openedx/core/djangoapps/theming/helpers.py

```diff
@@ -11,7 +11,7 @@
 from request_cache.middleware import RequestCache
 
 from microsite_configuration import microsite
```

Comment:
@mattdrayer we are importing microsite here to wrap microsite functions with theming functions. Let me know if you mean something else.